src/Datadog.Trace.ClrProfiler.Managed/Integrations/AspNetWebApi2Integration.cs

```diff
@@ -148,10 +150,16 @@ private static void UpdateSpan(dynamic controllerContext, Span span)
                 {
                 }
 
-                string resourceName = $"{method} {rawUrl}";
+                string resourceName = $"{method} {absoluteUri.ToLowerInvariant()}";
```

Comment:
Similar to [this comment](https://github.com/DataDog/dd-trace-dotnet/pull/361/files?file-filters%5B%5D=.cs#r288123436), we should try to replace `{controller}` and `{action}` in the route template with real values.